internal/release/promote/interactive_prompt_provider.go

```diff
@@ -72,6 +72,9 @@ func (i *InteractivePromptProvider) SelectTargetEnvironment(environments []*v1al
 }
 
 func (i *InteractivePromptProvider) SelectReleases(list cross.ReleaseList, maxColumnWidth int) (cross.ReleaseList, error) {
+	i.println()
+	i.printf(style.Notice("* indicates changes beyond just the version number\n\n"))
```

Comment:
Why include the trailing `\n\n` but not the leading one?
```suggestion
	i.printf(style.Notice("\n* indicates changes beyond just the version number\n\n"))
```